packages/cluster-ui/src/statementsPage/statementsPage.tsx

```diff
@@ -265,6 +273,20 @@ export class StatementsPage extends React.Component<
               </Text>
             </Dropdown>
           </PageConfigItem>
+          <PageConfigItem>
+            <div>
+              <input
+                type="checkbox"
+                id="full-table-scan-toggle"
+                defaultChecked={this.state.fullScan}
+                onChange={this.fullScanChange}
+              />
+              <label htmlFor="full-table-scan-toggle">
+                &nbsp;&nbsp;Only show statements that contain queries with full
```

Comment:
This is a _React thing_, but if you want to add literal spaces in HTML you can do something like 

```suggestion
                {"  "} Only show statements that contain queries with full
```

instead of using `&nbsp;`